splinepy/multipatch.py

```diff
@@ -0,0 +1,358 @@
+        -------
+        None
+        """
+        # Get minimum boundary id
+        max_BID = self.interfaces.min()
+
+        if boundary_id is None:
+            new_BID = max_BID - 1
+            log.debug(f"Creating new boundary with ID {abs(new_BID)}")
+        else:
+            # Make sure its negative
+            new_BID = -abs(int(boundary_id))
+            if new_BID < max_BID:
+                log.debug(f"Creating new boundary with ID {abs(new_BID)}")
+            else:
+                log.debug(
+                        "Adding new boundary elements to existing "
+                        f"boundary new boundary with ID {abs(new_BID)}"
+                )
+
+        try:
+            old_indices = self.interfaces[spline_ids, boundary_faces]
+
+            # Check if all old indices are negativ
+            if (old_indices < 0).all():
+                self.interfaces[spline_ids, boundary_faces] = new_BID
+            else:
+                raise ValueError(
+                        "One or more of the assigned boundary elements do not"
+                        " ly on the patch surface, please check topology"
+                )
+        except BaseException:
+            raise ValueError(
+                    "spline_ids and boundary_faces need to be one-dimensional."
+                    "\nIf this error proceeds please check if interfaces "
+                    "exists by calling, \nprint(<>.interfaces)"
+            )
+
+    @property
+    def para_dim(self):
+        """
+        Parametric dimension of the splineset
+        """
+        if self.splines is None:
+            raise ValueError("No Splines provided")
+        return self.splines[0].para_dim
+
+    @property
+    def dim(self):
+        """
+        Physical dimension of the splineset
+        """
+        if self.splines is None:
+            raise ValueError("No Splines provided")
+
+        return self.splines[0].dim
+
+    @property
+    def spline_boundary_centers(self):
+        """
+        This property function is a placeholder for once i figure out how to
+        use tracked-arrays
+
+        talk to at @j042
+        """
+        # If spline list is empty will throw excetion
+        return np.vstack([s.evaluate_boundary_centers for s in self.splines])
+
+    def determine_interfaces(self):
+        """
+        Retrieve interfaces info
+
+        Stores new informaton as interfaces
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        None
+        """
+        from splinepy.splinepy_core import interfaces_from_boundary_centers
+        from splinepy import settings
+
+        # Using the property instead of the the member, all necessery
+        # checks will be performed
+        self.interfaces = interfaces_from_boundary_centers(
+                self.spline_boundary_centers, settings.TOLERANCE, self.para_dim
+        )
+
+        log.debug("Successfully provided new interfaces using uff algorithm")
```

Comment:
```suggestion
        log.debug("Successfully provided new interfaces using uff algorithm")
        
        return self.interfaces
```
If this func is available, I'd expect it to return the determined interfaces